results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+- **Having an interface exclusively for coarse grain navigation.** While navigation may still be improved within other areas of JupyterLab (like the command palette recommendation above), navigation is critical enough to warrant its own attention. Adding a central location to change keyboard focus between the menu bar, main document area, status bar, both side bars, and possibly Settings would remove the most major keyboard navigation issues that appeared in this series of tests. In past versions of JupyterLab, this idea has been touched upon with the skiplink—a prompt that allows users to move focus immediately to the main document area if they begin tabbing from the start of JupyterLab. 
+
+#### More UI customization 
+
+This includes the discoverability of existing UI customization features and the addition of more or more detailed customization options. It is important to note that UI customization is most likely to support disabled users who still have some amount of vision available to them which was the case with this test’s participants.
+
+- **JupyterLab does have existing UI customization features by default.** Light versus dark themes, editor font size, and layout modes (Simple Mode, Presentation Mode) all allow users choices to set up JupyterLab in a way that may work better for them. Because these features require users to navigate multiple areas of JupyterLab to get to and do not require any domain-specific knowledge, the test script asked participants to engage with some of these features. The overwhelming response was one of positive surprise. Participants wanted to use these features and did not know they existed. Light/dark themes were the most likely to be known, while font size adjustments weren’t known by any participant. Screen magnifier users and other low-vision participants emphasized how helpful font size adjustment would be for them and that they wished they knew about them before.
+- **JupyterLab’s default—meaning without the addition of extensions—UI customization capabilities are limited.** In conversation with participants and in observing the issues they faced, we’ve determined additional UI customization features that would help them better use JupyterLab. Entirely new customization options include selecting from multiple syntax highlighting themes, being able to select major button colors (like the `--jp-brand-color` family), being able to select state colors (like the warning, error, and info color families), and to set a maximum width for documents in the main document area. Additions to existing customization options include more themes beyond light and dark modes, to be able to change all text sizes in JupyterLab, to be able to select text sizes directly rather than in opaque increments (ie. to set font sizes to a point, pixel, or percent value rather than selecting menu items with no feedback), and to assign other or additional indicators for the status of running or freshly completed code cells.
```

Comment:
```suggestion
- **JupyterLab’s default—meaning UI customization capabilities are limited without adding extensions.** In conversation with participants and observing their issues, we’ve determined additional UI customization features that would help them better use JupyterLab. Entirely new customization options include selecting from multiple syntax highlighting themes, being able to select major button colors (like the `--jp-brand-color` family), being able to select state colors (like the warning, error, and info color families), and to set a maximum width for documents in the main document area. Additions to existing customization options include more themes beyond light and dark modes, the ability to change all text sizes in JupyterLab, the ability to select text sizes directly rather than in opaque increments (i.e., to set font sizes to a point, pixel, or percent value rather than selecting menu items with no feedback), and to assign other or additional indicators for the status of running or freshly completed code cells.
```

Reply:
```suggestion
- **JupyterLab’s default UI customization capabilities are limited without adding extensions.** In conversation with participants and observing their issues, we’ve determined additional UI customization features that would help them better use JupyterLab. Entirely new customization options include selecting from multiple syntax highlighting themes, being able to select major button colors (like the `--jp-brand-color` family), being able to select state colors (like the warning, error, and info color families), and to set a maximum width for documents in the main document area. Additions to existing customization options include more themes beyond light and dark modes, the ability to change all text sizes in JupyterLab, the ability to select text sizes directly rather than in opaque increments (i.e., to set font sizes to a point, pixel, or percent value rather than selecting menu items with no feedback), and to assign other or additional indicators for the status of running or freshly completed code cells.
```

Above suggestion with one typo fixed for easy commit.